plugins/modules/onepassword_info.py

```diff
@@ -19,7 +19,8 @@
 requirements:
     - C(op) 1Password command line utility. See U(https://support.1password.com/command-line/)
 notes:
-    - Tested with C(op) version 0.5.5
+    - Only supports C(op) version 2 and later
```

Comment:
This looks like a breaking change that cannot be merged without a sufficient deprecation time.

Reply:
Yes, it is a breaking change. I was hoping to be able to reuse the work done for the lookup plugin with this module, but the error handling in the controller code relied heavily on `AnsibleLookupError` and it wasn't feasible to retool that to be used for both module and controller.

I can work on making this module work with both v1 and v2, but it will take some time. I wasn't sure if it was worth it for the module or if most people using this module have already updated to v2 and aren't concerned with v1 support.

Reply:
I cannot say whether it's worth it, but we do have a policy against breaking changes without deprecation, so as long as the v1 client didn't stop working completely (which I guess it didn't yet) we cannot just drop support for it.